astropy/extern/_strptime.py

```diff
@@ -38,7 +38,7 @@ def _getlang():
     # Figure out what the current language is set to.
     return locale.getlocale(locale.LC_TIME)
 
-class LocaleTime(object):
```

Comment:
I think it would be better to omit the `extern` files from the update.

Reply:
extern should definitely be left alone for any updates, as we try to simply copy paste their released versions as is for to have a good reference as ease of maintenance.